doc/api/dns.md

```diff
@@ -59,8 +59,21 @@ the [Implementation considerations section][] for more information.
 added: v0.11.3
 -->
 
-Returns an array of IP address strings that are being used for name
-resolution.
+Returns an array of IP address strings, formatted according to [rfc3986][],
+that are currently configured for DNS resolution. A string will include a port
+section if a custom port is used.
+
+eg.
+
+<!-- eslint-disable -->
+```js
+[
+  '4.4.4.4',
+  '[2001:4860:4860::8888]',
```

Comment:
Need to remove `[]` for this line.
Ref: https://github.com/nodejs/node/issues/13795